forest/shared/src/cli/mod.rs

```diff
@@ -175,6 +178,17 @@ impl CliOpts {
     }
 }
 
+/// convert `BigInt` to size string using byte size units (i.e. KiB, GiB, PiB, etc)
+/// Provided number cannot be negative, otherwise the function will panic.
+pub fn to_size_string(input: &BigInt) -> anyhow::Result<String> {
+    let bytes = u128::try_from(input)
+        .map_err(|e| anyhow::anyhow!("error parsing the input {}: {}", input, e))?;
+
+    Ok(Byte::from_bytes(bytes)
+        .get_appropriate_unit(true)
+        .to_string())
+}
+
```

Comment:
If you move a function around, make sure you move also its unit tests so they stay in the same module. These tests should also be moved https://github.com/ChainSafe/forest/blob/9d6180bb7b9faf14cd4fa9fed88b71b81ae1e034/forest/cli/src/cli/mod.rs#L216-L247

Reply:
Ah, I didn't see the tests. Good catch